runtime/lua/vim/lsp/handlers.lua

```diff
@@ -1,8 +1,7 @@
 local log = require('vim.lsp.log')
 local protocol = require('vim.lsp.protocol')
 local util = require('vim.lsp.util')
-local vim = vim
```

Comment:
This is harmless so i'd lean towards keeping it, not actively removing it. It can remain a "judgement call" in the sense that we don't need to proclaim it as a requirement in every file.

Reply:
Okay, I'll revert it.